grails-app/domain/au/org/ala/ecodata/ExternalId.groovy

```diff
@@ -13,7 +13,7 @@ class ExternalId implements Comparable {
     enum IdType {
         INTERNAL_ORDER_NUMBER, TECH_ONE_CODE, WORK_ORDER, GRANT_AWARD, GRANT_OPPORTUNITY, RELATED_PROJECT,
         MONITOR_PROTOCOL_INTERNAL_ID, MONITOR_PROTOCOL_GUID, TECH_ONE_CONTRACT_NUMBER, TECH_ONE_PARTY_ID, MONITOR_PLOT_GUID,
-        MONITOR_PLOT_SELECTION_GUID, MONITOR_MINTED_COLLECTION_ID, UNSPECIFIED }
+        MONITOR_PLOT_SELECTION_GUID, MONITOR_MINTED_COLLECTION_ID, UNSPECIFIED, RAID }
```

Comment:
This isn't a big deal, but it would be easier to understand this one if the acronym was expanded or prexfixed with NESP or something?